ui/plugins/controller.js

```diff
@@ -22,6 +22,8 @@ treeherder.controller('PluginCtrl', [
         $scope.job = {};
         $scope.artifacts = {};
 
+        var reftestUrlRoot = "http://hg.mozilla.org/mozilla-central/raw-file/tip/layout/tools/reftest/reftest-analyzer.xhtml#logurl=";
```

Comment:
Nothing appeared to break when I switched from http to https, so made that change. Dunno if it was http for some other reason, though.